CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 - Use `multibase` instead of `bs58` for base58 encoding. See [PR 56].
 
+- Create `protocol_stack` for Multiaddr. See [PR 60] 
```

Comment:
`multiaddr` `v0.15.0` just released, thus this needs to be moved to a new section for `v0.16.0` and the `Cargo.toml` in this project needs to be bumped.

Sorry for the inconvenience @John-LittleBearLabs.